[BUG] lint.sh breaks on charts with dependencies

Please see this CircleCI run: https://circleci.com/gh/kubernetes/charts/1156

The chart (chartmuseum) has a dependency on the common chart, and lint no likey:
```
Runing helm lint on the chart at incubator/chartmuseum
==> Linting incubator/chartmuseum
[ERROR] templates/: render error in "chartmuseum/templates/service.yaml": template: chartmuseum/templates/service.yaml:4:11: executing "chartmuseum/templates/service.yaml" at <include "common.full...>: error calling include: template: no template "common.fullname" associated with template "gotpl"

Error: 1 chart(s) linted, 1 chart(s) failed
```

I fixed this problem a while ago, but it appears that the CI scripts run have changed since?
https://github.com/kubernetes/charts/pull/2313

--- a/test/circle/install.sh
+++ b/test/circle/install.sh
@@ -22,6 +22,12 @@ sudo mv linux-amd64/helm /usr/local/bin
 rm -f helm-${HELM_LATEST_VERSION}-linux-amd64.tar.gz
 rm -rf linux-amd64
 
+# Setup Helm so that it will work with helm dep commands. Only the client
+# needs to be setup. In addition, the incubator repo needs to be
+# available for charts that depend on it.
+helm init -c
+helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+
 # Install A YAML Linter
 # Pinning to a version for consistency
 sudo pip install yamllint==1.8.1

--- a/test/circle/lint.sh
+++ b/test/circle/lint.sh
@@ -115,7 +115,10 @@ for directory in ${CHANGED_FOLDERS}; do
   if [ "$directory" == "incubator/common" ]; then
     continue
   elif [ -d $directory ]; then
-    printf "\nRuning helm lint on the chart at ${directory}\n"
+    printf "\nRunning helm dep build on the chart at ${directory}\n"
+    run helm dep build ${directory}
+
+    printf "\nRunning helm lint on the chart at ${directory}\n"
     run helm lint ${directory}
 
     yamllinter ${directory}

